File: C#/HttpClient.cs
using System;
using System.Net.Http;
using System.Threading.Tasks;

class Program
{
    static async Task Main(string[] args)
    {
        HttpClient client = new HttpClient();
        var response = await client.GetAsync("https://api.apitube.io/v1/news/everything?per_page=50&api_key=YOUR_API_KEY");

        if (response.IsSuccessStatusCode)
        {
            string responseBody = await response.Content.ReadAsStringAsync();
            Console.WriteLine(responseBody);
        }
        else
        {
            Console.WriteLine($"Error: {response.StatusCode}");
        }
    }
}

File: Dart/http_client.dart
import 'dart:convert';
import 'dart:io';

Future<void> main() async {
  final client = HttpClient();

  try {
    final uri = Uri.parse(
      'https://api.apitube.io/v1/news/everything?per_page=50&api_key=YOUR_API_KEY',
    );

    final request = await client.getUrl(uri);
    final response = await request.close();

    if (response.statusCode == 200) {
      final body = await response.transform(utf8.decoder).join();
      final data = jsonDecode(body);
      print(JsonEncoder.withIndent('  ').convert(data));
    } else {
      print('HTTP Request Error: ${response.statusCode}');
    }
  } catch (e) {
    print('Error: $e');
  } finally {
    client.close();
  }
}
